cmd/runstatus_maintenance_add.go

```diff
@@ -83,15 +83,39 @@ var runstatusMaintenanceAddCmd = &cobra.Command{
 			}
 		}
 
-		startDate, err := pickupDatePrompt(reader, "Choose start date:")
+		var startDate time.Time
+		sDate, err := cmd.Flags().GetString("start-date")
 		if err != nil {
 			return err
 		}
+		if sDate == "" {
+			startDate, err = pickupDatePrompt(reader, time.Now(), "Choose start date:")
+			if err != nil {
+				return err
+			}
+		} else {
+			startDate, err = time.Parse("2006-01-02T15:04:05-0700", sDate)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
you may move the `if err != nil { return err }` after the big `if sDate`.

Reply:
thx

Reply:
It's changed but github don't show it !!!